service/core/serverObj/v2ray.go

```diff
@@ -228,21 +234,35 @@ func (v *V2Ray) Configuration(info PriorInfo) (c Configuration, err error) {
 				},
 			}
 		case "vless":
+			security := v.Security
+			if security == "" {
+				security = "auto"
+			}
+			flow := v.Flow
+			if flow == "" {
+				flow = "xtls-rprx-direct"
+			}
 			core.Settings.Vnext = []coreObj.Vnext{
 				{
 					Address: v.Add,
 					Port:    port,
 					Users: []coreObj.User{
 						{
-							ID:         id,
 							Encryption: "none",
+							Flow:       flow,
+							ID:         id,
+							Level:      8,
+							Security:   security,
 						},
 					},
 				},
 			}
-		}
-		core.StreamSettings = &coreObj.StreamSettings{
```

Comment:
Why removing?

Reply:
We already have it on line 206:

```
core.StreamSettings = &coreObj.StreamSettings{
	Network: v.Net,
}
```

Do you want me to set it only to vless or vmess could have it?

Reply:
Thanks. I'll have a double check later.

Reply:
Okay let me know if we need to change any.